tracer/src/Datadog.AutoInstrumentation.NativeLoader/runtimeid_store.cpp

```diff
@@ -3,6 +3,7 @@
 #include "util.h"
 
 #include "../../../shared/src/native-src/util.h"
+using namespace datadog::shared::nativeloader;
```

Comment:
why do you use the `using statement` which will copy every symbols from `datadog::shared::nativeloader` in the current compilation unit.
Did you mean `namespace datadog::shared::nativeloader {` ?
Or am I missing something ?

Reply:
Fixed in ebde2f57c8ef03fd167862f981abd5c4958c222a